llvm/include/llvm/Support/DXILABI.h

```diff
@@ -99,6 +101,10 @@ enum class SamplerFeedbackType : uint32_t {
 const unsigned MinWaveSize = 4;
 const unsigned MaxWaveSize = 128;
 
+LLVM_ABI ArrayRef<EnumEntry<llvm::dxil::ResourceClass>> getResourceClasses();
```

Comment:
No need to fully qualify this
```suggestion
LLVM_ABI ArrayRef<EnumEntry<ResourceClass>> getResourceClasses();
```